polaris-react/src/components/ResourceItem/ResourceItem.tsx

```diff
@@ -300,46 +296,31 @@ class BaseResourceItem extends Component<CombinedProps, State> {
     }
 
     const containerMarkup = (
-      <UseFeatures>
-        {({polarisSummerEditions2023}) => (
-          <Box
-            id={this.props.id}
-            position="relative"
-            padding="3"
-            paddingInlineStart={
-              polarisSummerEditions2023 ? '3' : {xs: '4', sm: '5'}
-            }
-            paddingInlineEnd={
-              polarisSummerEditions2023 ? '3' : {xs: '4', sm: '5'}
-            }
-            zIndex="var(--pc-resource-item-content-stacking-order)"
+      <Box
+        id={this.props.id}
+        position="relative"
+        paddingInlineStart={itemPaddingInline}
+        paddingInlineEnd={itemPaddingInline}
+        paddingBlockStart={itemPaddingBlock}
+        paddingBlockEnd={itemPaddingBlock}
+        zIndex="var(--pc-resource-item-content-stacking-order)"
+      >
+        <HorizontalGrid columns={{xs: '1fr auto'}}>
+          <HorizontalGrid
+            columns={{xs: media || selectable ? 'auto 1fr' : '1fr'}}
+            gap={gapBetweenOwnedAndChildren}
           >
-            <HorizontalGrid columns={{xs: '1fr auto'}}>
-              <HorizontalGrid
-                columns={{xs: media || selectable ? 'auto 1fr' : '1fr'}}
-                gap={polarisSummerEditions2023 ? '3' : '5'}
-              >
-                {ownedMarkup}
-                <HorizontalStack
-                  gap={polarisSummerEditions2023 ? '3' : '4'}
```

Comment:
This stack will always have only one child (the `<Box>`), so I removed the unnecessary `gap` prop.